src/libs/actions/PaymentMethods.ts

```diff
@@ -109,74 +122,80 @@ function getMakeDefaultPaymentOnyxData(bankAccountID, fundID, previousPaymentMet
     return onyxData;
 }
 
+type MakeDefaultPaymentMethodParams = {
```

Comment:
You can move params type to function body (examples in Fabio's [PR](https://github.com/Expensify/App/pull/28269))

context:
![image](https://github.com/Expensify/App/assets/46095609/4d80e0df-8245-4597-aab7-206c32c8b97c)

Reply:
Thanks, updated